mitodl/mitxpro
# min_weeks and max_weeks field added to replace duration field

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5997

### Description (What does it do?)
This PR adds two new fields in Course and Program
1. min_weeks
2. max_weeks
And they will eventually remove/replace `duration` field

### Screenshots (if appropriate):
![image](https://github.com/user-attachments/assets/fb1c93d3-092a-4a2c-822c-ddf4adb67b6f)
![image](https://github.com/user-attachments/assets/af98b24d-f23b-414f-af08-a62b54b39bf9)


### How can this be tested?
1. Run Migrations
2. Go to
    a. /api/courses
    b. /api/programs

And check for min_weeks and max_weeks fields
